src/typechecker.erl

```diff
@@ -416,13 +411,13 @@ glb(T1, T2, A, TEnv) ->
         %% actual use case.
         true -> {type(none), constraints:empty()};
         false ->
-            case gradualizer_cache:get_glb(TEnv#tenv.module, T1, T2) of
+            case gradualizer_cache:get_glb(maps:get(module, TEnv), T1, T2) of
```

Comment:
If we want to avoid maps:get/2 here, we could match it out as `TEnv = #{module := Module}` in the function head... (but I'm not saying maps:get/2 is a problem).

Reply:
Yes, I'm aware of it. I think it's fine the way it is.